ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -149,14 +145,26 @@ class WalletHomeScreen extends React.Component<Props, never> {
     // WIP loadTransactions should not be called from here
     // (transactions should be persisted & fetched periodically)
     // WIP WIP create pivotal story
-    this.props.loadCards();
     this.props.loadTransactions();
   }
 
-  public render(): React.ReactNode {
-    const showCards = this.props.cardsNumber > 0;
+  // check the cards to display (none, one or two cards)
+  private getCardType(): CardType {
+    const cards = this.props.cards;
 
-    // TODO: cards list is currently mocked, will be implemented properly @https://www.pivotaltracker.com/story/show/157422715
+    switch (this.props.cards.length) {
+      case 0:
+        return { type: CardEnum.NONE };
+      case 1:
+        return { type: CardEnum.HEADER, card: cards[0] };
+      default:
+        return { type: CardEnum.FAN, cards: [cards[0], cards[1]] };
```

Comment:
According to @matteodesanti 's clarifications, if 3+ cards exist, they should all be listed in this component -- so this switch...case approach probably isn't for the best.